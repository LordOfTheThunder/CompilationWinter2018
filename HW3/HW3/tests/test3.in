void TestInt(bool a) {
	int k = 133;
}
void TestIntMismatchByte(bool a) {
	int k = 133b;
}

void main() { print("Max is cool"); }
