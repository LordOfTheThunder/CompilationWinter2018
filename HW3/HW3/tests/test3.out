---end scope---
a BOOL -1
k INT 0
---end scope---
a BOOL -1
k INT 0
---end scope---
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
TestInt (BOOL)->VOID 0
TestIntMismatchByte (BOOL)->VOID 0
main ()->VOID 0
